### PR TITLE
New frontend #72

### LINKED ISSUE #72: GUI(-dev): Set-up new frontend
The current frontend is not maintainable anymore because it is blown-up with too much dependencies which are mostly deprecated.
Therefore, the better solution is to set-up a fresh and new frontend application.

- use ngx-admin template
- use up-to-date modules
  - Angular 5.x
  - ngx-admin newest version
  - other RELEVANT modules
- ONLY use relevant modules
- set-up correctly
  - webpack/other config.
  - configure `npm scripts` correctly
- get it working
- move the old project into a folder like `gui-dev-old` in the `GUI-dev` folder when finished
  - needed to later port old functionalities into new application set-up

### PR DESCRIPTION
**Pull request by deKilla**
resolves #72 

added:
complete GUI replacement: new GUI based on ngx-admin
- located in the GUI-dev folder
- uses the starter-kit (clean implementation)

changed:
GUI-dev archived as GUI-dev-old

### CODE REVIEW

File: GUI-dev-old/package.json [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    "webpack-dashboard": "^0.1.8",
+    "webpack-dev-middleware": "^1.6.1",
+    "webpack-dev-server": "^2.1.0-beta.9",
+    "webpack-md5-hash": "^0.0.5",
+    "webpack-merge": "^0.15.0",
+    "compression-webpack-plugin": "^0.3.1",
+    "es6-promise": "^3.1.2",
+    "es6-shim": "^0.35.0",
+    "es7-reflect-metadata": "^1.6.0",
+    "electron": "^1.4.0"
```
Comment (amarbajric) [inline GUI-dev-old/package.json:104]: Do we need this?
Please check starter-kit etc. if we need this package.
In general, check the whole `package.json` again and remove scripts and npm modules if they are not needed.

**Comment by amarbajric** [comment]
**REVIEW**
- add description to your PR please what you have done.
  - you can take the CHANGELOG entry and add it here
- enhance your CHANGELOG entry
  - i.e. "added new frontend project in the GUI-dev folder etc."
  - just a little bit more info to get a better feel what has been done. nothing more
- check `package.json` and `other files` again and check if they are really relevant?

File: GUI-dev-old/package.json (hunk 1 above)
Comment (deKilla) [reply GUI-dev-old/package.json:104]: GUI-dev-old is just the old GUI archived in order to make room for the new ngx-admin. The content of the old GUI was not altered as part of this issue.

Comment (amarbajric) [reply GUI-dev-old/package.json:104]: never mind this was my fault...wrong folder 😆

**Comment by amarbajric** [comment]
Thank you for setting up the new frontend.
Everything looks fine to me for now.
Changes, further clean-up and other issues will be resolved during implementation of the frontend itself.

**Review by amarbajric — APPROVED** [review]

**Pull request merged**